action-server/covidflow/actions/constants.py

```diff
@@ -15,17 +18,30 @@
     LAST_HAS_DIFF_BREATHING_SLOT,
 ]
 
-
+### Assessment
 FEEL_WORSE_SLOT = "feel_worse"
 SYMPTOMS_SLOT = "symptoms"
 HAS_FEVER_SLOT = "has_fever"
 HAS_COUGH_SLOT = "has_cough"
 HAS_DIFF_BREATHING_SLOT = "has_diff_breathing"
 
-
+### Profile
+LANGUAGE_SLOT = "language"
 FIRST_NAME_SLOT = "first_name"
+PHONE_NUMBER_SLOT = "phone_number"
 PRECONDITIONS_SLOT = "preconditions"
 HAS_DIALOGUE_SLOT = "has_dialogue"
 PROVINCE_SLOT = "province"
 AGE_OVER_65_SLOT = "age_over_65"
-PROVINCIAL_811_SLOT = "provincial_811"
+
+## Slots not used by DB
```

Comment:
Slots used only in flows maye? It's always weird to define something based on what it is not